util/util.go

```diff
@@ -1014,3 +1014,19 @@ func VerifySnapshotLabels(labels map[string]string) error {
 	}
 	return nil
 }
+
+func IsTagsEqual(tags1, tags2 []string) bool {
+	if !sort.StringsAreSorted(tags1) {
+		logrus.Warn("BUG: Tags are not sorted, sorting now")
```

Comment:
Make this as a debug log. (we are changing the log level to info, so this kind of log can be in debug, so it will not be shown by default until the level is changed to debug.)